assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/edit.tsx

```diff
@@ -59,6 +62,7 @@ export const Edit = ( {
 						} }
 					/>
 				) }
+				<CartSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 45).